Controller/Annotations/Route.php

```diff
@@ -22,9 +22,8 @@ class Route extends BaseRoute
     public function __construct(array $data)
     {
         parent::__construct($data);
-        $requirements = $this->getRequirements();
-        $requirements['_method'] = $this->getMethod();
-        $this->setRequirements($requirements);
+
+        $this->setMethods((array) $this->getMethod());
```

Comment:
I guess this change is problematic, since we override the method, without checking if something was set before .. ?